ruby_on_rails/configure_cd.md

```diff
@@ -1,23 +1,41 @@
 # Configure the Continuous Deployment (CD)
 
-You can configure continuous deployment on SemaphoreCI.
+We use Semaphore to deploy most of our apps. Some projects use CircleCI and
+mobile apps use Fastlane. This article is about how to deploy to Heroku with
+Semaphore.
 
-Configure an automatic deployment to each Heroku application (`[project-name]-master`,`[project-name]-develop`,
-`[project-name]-testing`). Use the API key of `admin@renuo.ch` for this task. wg-operations will help you, if you do
-not have access.
+## Prerequisites
 
-The deployment should execute the following deploy commands:
+* Heroku API key of `admin@renuo.ch` (*wg-operations* can help with that)
+
+## Setup `master` and `develop`
+
+Add two servers (very tiny plus button) and name them `[project-name]-master`
+and `[project-name]-develop`.
```

Comment:
I propose to name the servers only `master` and `develop`. the project name is already given by the current scope (you are in the project)